### PR TITLE
Move out user and clean dependencies

### PR DESCRIPTION
**Pull request by xepozz**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ✔️
| Breaks BC?    | ✔️
| Fixed issues  | #275

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,20 +30,13 @@
...
         "yiisoft/http": "^1.0",
         "yiisoft/injector": "^1.0",
         "yiisoft/network-utilities": "^3.0@dev",
         "yiisoft/router": "^3.0@dev",
-        "yiisoft/security": "^1.0",
-        "yiisoft/serializer": "^3.0@dev",
-        "yiisoft/session": "^3.0@dev",
-        "yiisoft/validator": "^3.0@dev",
-        "yiisoft/yii-event": "^3.0@dev"
+        "yiisoft/validator": "^3.0@dev"
```
Comment (samdark) [inline composer.json:39]: yiisoft/user should be added

**Review by samdark — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (xepozz) [reply composer.json:39]: Why?

**Review by roxblnfk — APPROVED** [review]
I also think there is no need to add a `user` as as dependency

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Finished and merged. Thank you!

**Pull request merged**